src/EditModel/event-program/program-access/utils.js

```diff
@@ -1,33 +1,35 @@
 import { isEqual } from 'lodash/fp';
 
-export const areSharingPropertiesSimilar = (a, b) => {
-    if (a.publicAccess !== b.publicAccess) return false;
-    if (!!a.externalAccess !== !!b.externalAccess) return false;
+export const areSharingPropertiesSimilar = (modelA, modelB) => {
+    const sharingA = modelA.sharing;
+    const sharingB = modelB.sharing;
+    if (sharingA.public !== sharingB.public) return false;
+    if (!!sharingA.externalAccess !== !!sharingB.externalAccess) return false;
 
     const compareFunction = (a, b) => a.id < b.id;
     if (
         !isEqual(
-            Array.sort(a.userAccesses || [], compareFunction),
-            Array.sort(b.userAccesses || [], compareFunction),
+            Array.sort(sharingA.users || [], compareFunction),
```

Comment:
I have to double check this. Because this isn't actually an array anymore... I've tested it and it kind of works, but needs investigation... Also where does `Array.sort` come from?